src/sqs/sqs.ts

```diff
@@ -252,7 +252,7 @@ export class SqsQueue<MessagePayload> {
 			message,
 			payload,
 			log: listenerContext.log.child({
-				id: message.MessageId,
+				messageId: message.MessageId,
```

Comment:
renamed to messageID for clarity!

Reply:
Nice. Small change but makes those logs a lot easier to read.